Tests/StarknetTests/Resources/TypedData/typed_data_rev_0_validate_example.json

```diff
@@ -1,38 +1,83 @@
 {
   "types": {
     "StarkNetDomain": [
-      {"name": "name", "type": "string"},
-      {"name": "version", "type": "felt"},
-      {"name": "chainId", "type": "felt"}
+      {
```

Comment:
As I [mentioned](https://github.com/software-mansion/starknet.swift/pull/194#discussion_r1601297678) in another PR, I don't think format's necessary

Reply:
Correct, I've restored previous formatting.